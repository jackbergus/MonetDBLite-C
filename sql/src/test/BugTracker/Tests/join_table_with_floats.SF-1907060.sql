create table t2 (id float);
create table t1 (age float);

select age from t1, t2 where id < 0.0 limit 10;

drop table t1;
drop table t2;

