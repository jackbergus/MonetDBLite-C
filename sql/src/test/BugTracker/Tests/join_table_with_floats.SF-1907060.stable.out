stdout of test 'join_table_with_floats.SF-1907060` in directory 'src/test/BugTracker` itself:


# 00:15:27 >  
# 00:15:27 >   mserver5 "--config=/ufs/goncalve/scratch/MonetDB/Stable/MonetDB5/etc/monetdb5.conf" --debug=10 --set "monet_mod_path=/ufs/goncalve/scratch/MonetDB/Stable/MonetDB5/lib/MonetDB5:/ufs/goncalve/scratch/MonetDB/Stable/MonetDB5/lib/MonetDB5/lib:/ufs/goncalve/scratch/MonetDB/Stable/MonetDB5/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/goncalve/scratch/MonetDB/Stable/MonetDB5/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/goncalve/scratch/MonetDB/Stable/MonetDB5/var/MonetDB5/sql_logs"  --set mapi_open=true --set xrpc_open=true --set mapi_port=30709 --set xrpc_port=40808 --set monet_prompt= --trace  "--dbname=mTests_src_test_BugTracker" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 00:15:27 >  



# MonetDB server v5.4.1, based on kernel v1.22.1
# Serving database 'mTests_src_test_BugTracker'
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-2008 CWI, all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://amelia.ins.cwi.nl:30709/
# MonetDB/SQL module v2.22.1 loaded

Ready.
#function user.main():void;
#    clients.quit();
#end main;

Over..

# 00:15:27 >  
# 00:15:27 >  mclient -lsql -umonetdb -Pmonetdb --host=amelia --port=30709 
# 00:15:27 >  

% sys.t1 # table_name
% age # name
% double # type
% 22 # length

# 00:15:27 >  
# 00:15:27 >  Done.
# 00:15:27 >  

